### PR TITLE
HOTFIX: remove unused import

### PR DESCRIPTION
**Pull request by mjsax**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java [hunk 1]
```diff
@@ -48,13 +48,12 @@
 import org.apache.kafka.streams.test.OutputVerifier;
 import org.apache.kafka.test.TestUtils;
 import org.junit.After;
-import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
-import java.time.Duration;
```
Comment (mjsax) [inline streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java:56]: This is the actual hotfix

File: streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java [hunk 2]
```diff
@@ -64,13 +63,13 @@
...
 import java.util.Properties;
 import java.util.Set;
-import java.util.Arrays;
 import java.util.regex.Pattern;
 
 import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
```
Comment (mjsax) [inline streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java:73]: replace deprecated `Assert.assertThat` with `MatcherAssert.assertThat` as side cleanup.

**Comment by mjsax** [comment]
Call for review @guozhangwang @bbejeck @vvcephei @ableegoldman @cadonna 

I think this happened because of a bad cherry-pick from my side. Sorry for breaking the build... :(

**Review by bbejeck — APPROVED** [review]
LGTM

**Review by cadonna — APPROVED** [review]

**Pull request merged**